File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/service/notification/NotifyOfPlaybackEvents.java
package com.lasthopesoftware.bluewater.client.playback.service.notification;

import com.lasthopesoftware.bluewater.client.browsing.library.items.media.files.ServiceFile;

public interface NotifyOfPlaybackEvents {
	void notifyPlaying();
	void notifyPaused();
	void notifyStopped();
	void notifyPlayingFileChanged(ServiceFile serviceFile);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/service/notification/NotifyOfPlaybackEvents.kt
package com.lasthopesoftware.bluewater.client.playback.service.notification

interface NotifyOfPlaybackEvents {
    fun notifyPlaying()
    fun notifyPaused()
    fun notifyInterrupted()
    fun notifyStopped()
    fun notifyPlayingFileUpdated()
}
